diff --git a/node_modules/eslint-plugin-fsd-import/lib/rules/layer-imports.js b/node_modules/eslint-plugin-fsd-import/lib/rules/layer-imports.js
index 75153a5..b054e0c 100644
--- a/node_modules/eslint-plugin-fsd-import/lib/rules/layer-imports.js
+++ b/node_modules/eslint-plugin-fsd-import/lib/rules/layer-imports.js
@@ -38,8 +38,8 @@ module.exports = {
         const { alias = '', ignoreImportPatterns = [] } = context.options[0] ?? {};
 
         const layers = {
-            'app': ['pages', 'widgets', 'features', 'entities', 'shared'],
-            'pages': ['widgets', 'features', 'entities', 'shared'],
+            'app': ['screens', 'widgets', 'features', 'entities', 'shared'],
+            'screens': ['widgets', 'features', 'entities', 'shared'],
             'widgets': ['features', 'entities', 'shared'],
             'features': ['entities', 'shared'],
             'entities': ['entities', 'shared'],
@@ -50,7 +50,7 @@ module.exports = {
             'app': 'app',
             'entities': 'entities',
             'features': 'features',
-            'pages': 'pages',
+            'screens': 'screens',
             'widgets': 'widgets',
             'shared': 'shared',
         };
